stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/usecase/GetThreadTest.kt

```diff
@@ -1,62 +0,0 @@
-package io.getstream.chat.android.offline.usecase
-
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.nhaarman.mockitokotlin2.doReturn
-import com.nhaarman.mockitokotlin2.mock
-import com.nhaarman.mockitokotlin2.whenever
-import io.getstream.chat.android.client.utils.Result
-import io.getstream.chat.android.offline.ChatDomainImpl
-import io.getstream.chat.android.offline.channel.ChannelController
-import io.getstream.chat.android.offline.thread.ThreadController
-import io.getstream.chat.android.offline.utils.`should be equal to result`
-import io.getstream.chat.android.test.randomCID
-import io.getstream.chat.android.test.randomString
-import kotlinx.coroutines.runBlocking
-import org.amshove.kluent.`should throw`
-import org.amshove.kluent.`with message`
-import org.amshove.kluent.invoking
-import org.junit.Before
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4::class)
-internal class GetThreadTest {
```

Comment:
Not actual anymore. Tests weren't the best. They only check `validateCid` function